### ISSUE TITLE
request for PBMC pre-trained model and training data

### ISSUE THREAD

**Issue by LiaoYunxi** [elo-nsrb/Cellformer#3]
Hello, I would like to use the Cellformer tool, but I noticed that only the pre-trained model for AD data is available on GitHub. Could you please provide the pre-trained model from cellformer_pbmc? Additionally, the data folder lacks the training dataset and annotation file examples needed for model training, making it difficult for me to train a model on my own. Could you please provide example data or the training dataset and annotation files for PBMC data? Thank you!

**Comment by maalbadri** [comment]
@elo-nsrb Hi Eloise! Are you still planning on adding this to the repo?

**Comment by elo-nsrb** [comment]
Hello, I have added the weights of the top 3 models trained using PBMC data with a cross-validation scheme. They are located in the folder cellformer_pbmc. For the PBMC model, I used single-cell data from [GreenleafLab/ArchR_2020](https://github.com/GreenleafLab/ArchR_2020) to create the training set, and the cells were already annotated. I have also included an example of the input single-cell peak matrix.
